github-adapter/src/main/java/be/xplore/githubmetrics/githubadapter/config/GithubRestClientConfig.java

```diff
@@ -20,6 +30,8 @@ public class GithubRestClientConfig {
     private final ObservationRegistry observationRegistry;
     private final GithubRestClientRequestObservationConvention githubRestClientRequestObservationConvention;
 
+    private final Duration timeout = Duration.ofSeconds(15L);
```

Comment:
I wonder if this is not too short. I know that 15 secs is a lot but the Api felt always felt quite slow when making manual requests. Also there is no problem for us to wait a bit, its not that realtime

Reply:
fair point. bumped it to 30seconds